models/user.js

```diff
@@ -1,34 +1,23 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-const yellow = 'rgb(255, 215, 18)';
+module.exports = function(sequelize, Sequelize) {
+  var User = sequelize.define("users", {
+    username: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true
+    },
+    password: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true
```

Comment:
password doesn't need to be unique, since we would likely want to allow non-unique password from different users